docs(getting_started,fill): added solc building instructions for arm64 users with fill issues (#1217)
ethereum/execution-spec-tests: docs/getting_started/installation.md

If installed via `curl`, `uv` will download Python for your target platform if one of the required versions (Python 3.10, 3.11 or 3.12) is not available natively.

## Installation Commands

> Clone [execution-spec-tests](https://github.com/ethereum/execution-spec-tests) and install its dependencies. If you are using an ARM64 OS ensure that you select the ARM64 tab below.

Reviewer: ```suggestion
Clone [execution-spec-tests](https://github.com/ethereum/execution-spec-tests) and install its dependencies.
```

> === "x86-64"

Reviewer: ```suggestion
=== "All x86-64 platforms and ARM64 macOS"
```

    ```console
    git clone https://github.com/ethereum/execution-spec-tests
    cd execution-spec-tests
    uv sync --all-extras
    uv run solc-select use 0.8.24 --always-install
    ```

> === "ARM64"

Reviewer: ```suggestion
=== "ARM64, Linux"
```

    ```console
    git clone https://github.com/ethereum/execution-spec-tests
    cd execution-spec-tests
    uv sync --all-extras
    ```

>     Follow [this guide](./installation_troubleshooting.md#problem-exception-failed-to-compile-yul-source) to build solidity from source.

Reviewer: ```suggestion
    Follow [this guide](./installation_troubleshooting.md#problem-exception-failed-to-compile-yul-source) to build the `solc` binary from source.
```

>     Follow [this guide](./installation_troubleshooting.md#problem-exception-failed-to-compile-yul-source) to build the `solc` binary from source.

Reviewer: ```suggestion
    Then follow [this guide](./installation_troubleshooting.md#problem-exception-failed-to-compile-yul-source) to build the `solc` binary from source and copy it to the expected location.
```
